news/q3-2024-recap-from-rebecca-rumbul.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Q3 2024 Recap from Rebecca Rumbul
+byline: Rust Foundation Team
+description: >-
+  A review of the Rust Foundation's activities in July, August, and September of
+  2024.
+date: 2024-10-31T12:00:00Z
+tags:
+  - 'quarterly updates '
+  - foundation
+index: false
+layout: layouts/news.njk
+---
+I’m pleased to have a number of exciting achievements from the Foundation and staff team to reflect on in this quarterly recap for July, August, and September of 2024. Thank you for your patience with us in assembling this recap – following the flurry of activity in Q3, many members of our team took time off to regroup which delayed this post.
+
+## Summary
+
+During the third quarter of 2024, the Rust Foundation achieved several significant milestones. The Foundation convened the first meeting of the[<u>Safety Critical Consortium</u>](https://foundation.rust-lang.org/news/announcing-the-safety-critical-rust-consortium/), establishing the scope and interest areas for this important initiative. Four new members joined the Foundation, and substantial progress was made in software supply chain engineering, including advancements in crate provenance tracking and agreement on using The Update Framework (TUF) for crate signing and mirroring.
+
+The Foundation's growing influence was reflected in 15 organic media mentions across global industry outlets. Staff actively engaged with the community through speaking engagements at significant events, including [RustConf](https://rustconf.com/),[<u>Rust Global</u>](https://rustconf.com/programs/#rust_global),[<u>Open Source Congress Beijing</u>](https://www.opensourcecongress.org/), and[<u>Open Source Summit Vienna</u>](https://osseu2024.sched.com/).
+
+Notable achievements included:
+
+* Successfully onboarding a new Infrastructure Engineer,
+* Awarding 19 new fellowships for 2024-25, and
+* The continued progression of the Rust Specification work.
+
+## Q3 Goals
+
+During the July 2024 Board Meeting, we set several ambitious goals for the third quarter. These included awarding and onboarding 2024 Fellows, organizing RustConf and associated events, including Rust Global, holding an in-person board meeting, defining the C++/Rust interop strategy, and implementing Foundation-driven malicious crate detection tooling in GCP. Substantial staff time was devoted to RustConf 2024 organization and Community Grants Fellowship Awards.
+
+## HR & Administration
+
+In the third quarter, we saw good progress in HR and administration. Marco Ieni joined as Infrastructure Engineer on July 15, substantially impacting the Infrastructure Team's capacity. Plans were made for a week-long staff offsite in Iceland, focusing on team cohesion and future planning. The Board's People Committee began preparations for annual reviews, including the Executive Director's evaluation process.
+
+## Finance & Legal
```

Comment:
Let's take out this finance/legal section. I think we should simply share this in-context in the annual report